packages/components/package.json

```diff
@@ -19,7 +19,7 @@
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
   "scripts": {
     "build": "rollup --config",
-    "prepublishOnly": "rollup --config",
+    "prepublishOnly": "rollup --config && node scripts/check-compiled-css.mjs",
```

Comment:
could we use a command directly in the CLI? something like this?
```
"prepublishOnly": "rollup --config && test -f /path/to/file && [do something here, not sure what]
```

Reply:
if a bash one-liner is preferred, we can go with that – not overly opinionated